tutorials/01_Basic_QA_Pipeline.ipynb

```diff
@@ -224,188 +154,149 @@
-    "from haystack.nodes import BM25Retriever\n",
-    "\n",
-    "retriever = BM25Retriever(document_store=document_store)"
+    "from haystack.utils import clean_wiki_text, convert_files_to_docs\n",
+    "docs = convert_files_to_docs(\n",
+    "    dir_path=doc_dir,\n",
+    "    clean_func=clean_wiki_text,\n",
+    "    split_paragraphs=True\n",
+    ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "3. Write these Documents into the DocumentStore."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "pycharm": {
-     "is_executing": false,
-     "name": "#%%\n"
-    }
-   },
    "outputs": [],
    "source": [
-    "# Alternative: An in-memory TfidfRetriever based on Pandas dataframes for building quick-prototypes with SQLite document store.\n",
-    "\n",
-    "# from haystack.nodes import TfidfRetriever\n",
-    "# retriever = TfidfRetriever(document_store=document_store)"
-   ]
+    "# Now, let's write the dicts containing documents to our DB.\n",
+    "document_store.write_documents(docs)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "While the default code in this tutorial uses Game of Thrones data, you can also supply your own. So long as your data adheres to the [input format](https://docs.haystack.deepset.ai/docs/document_store#input-format) or is cast into a [Document object](https://docs.haystack.deepset.ai/docs/documents_answers_labels#document), it can be written into the DocumentStore."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Reader\n",
-    "\n",
-    "A Reader scans the texts returned by retrievers in detail and extracts the k best answers. They are based\n",
-    "on powerful, but slower deep learning models.\n",
-    "\n",
-    "Haystack currently supports Readers based on the frameworks FARM and Transformers.\n",
-    "With both you can either load a local model or one from Hugging Face's model hub (https://huggingface.co/models).\n",
-    "\n",
-    "**Here:** a medium sized RoBERTa QA model using a Reader based on FARM (https://huggingface.co/deepset/roberta-base-squad2)\n",
+    "## Initializing the Retriever\n",
     "\n",
-    "**Alternatives (Reader):** TransformersReader (leveraging the `pipeline` of the Transformers package)\n",
-    "\n",
-    "**Alternatives (Models):** e.g. \"distilbert-base-uncased-distilled-squad\" (fast) or \"deepset/bert-large-uncased-whole-word-masking-squad2\" (good accuracy)\n",
-    "\n",
-    "**Hint:** You can adjust the model to return \"no answer possible\" with the no_ans_boost. Higher values mean the model prefers \"no answer possible\"\n",
-    "\n",
-    "#### FARMReader"
+    "Initialize the `BM25Retriever`. For more Retriever options, see [Retriever](https://docs.haystack.deepset.ai/docs/retriever)"
```

Comment:
I liked the way the original tutorial briefly explained what Retrievers do. 

I'm a person that always starts from tutorials rather than the docs, and this step would not provide enough information to understand what's happening in the block below, except from "we're creating an object, go read the docs to know why". One or two lines with the gist of it would go a long way imho.

PS: the same goes for the Reader. The DocumentStore is a bit more self-explanatory, but to be coherent we could add a brief intro for it too. One line is really enough I think.

Reply:
Ok good to hear your opinion on this. I think old tutorials had a lot of explanation, and probably too much. In the diataxis framework, they advocate for no conceptual explanation in tutorials. I think we can find a good middle ground between the two.

Reply:
Agree! The old tutorials had too much explanation in my opinion too, so it's more about not going too much into the opposite direction.